overlord/snapstate/snapstate.go

```diff
@@ -365,6 +365,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		}
 	}
 
+	// Implicitly set --unaliased flag for parallel installs to avoid
+	// alias conflicts with the main snap
+	if snapsup.InstanceKey != "" {
+		snapsup.Unaliased = true
+	}
```

Comment:
as we mainly we want to affect installs, as refreshes should follow the current state of things, we usually put this kind of logic in ensureInstallPreconditions I think

Reply:
Thanks for pointing this out, Updated!